bitcoin/script/miniscript.h

```diff
@@ -514,22 +514,22 @@ struct Node {
                 case NodeType::RIPEMD160: return CScript() << OP_SIZE << 32 << OP_EQUALVERIFY << OP_RIPEMD160 << node.data << (verify ? OP_EQUALVERIFY : OP_EQUAL);
                 case NodeType::HASH256: return CScript() << OP_SIZE << 32 << OP_EQUALVERIFY << OP_HASH256 << node.data << (verify ? OP_EQUALVERIFY : OP_EQUAL);
```

Comment:
Would it make sense to use `BuildScript` for everything, also the cases that don't involve nested scripts?

Reply:
I think so. I'm adding a commit replacing all `CScript()` calls with `BuildScript()` ones.